Fixed typos and chars

js-denton opened the pull request:
Были добавлены улучшения относящиеся к синтаксису и структуре самого текста в md формате.

js-denton committed:
Fixed typos

js-denton committed:
Removed a duplicate section

js-denton committed:
Removed redundant word

js-denton committed:
Broke to lines paragraphs

ligurio commented on 10_software_quality.md:
```diff
@@ -416,7 +416,7 @@ Quality Assurance Plans”.
 образом, в виде соответствующих "подтем" в рамках формата SWEBOK, так как,
 будучи тесно связанными и взаимодополняющими, проверка и аттестация все же
 обладают и самостоятельным содержанием), проверка (верификация) и аттестация –
-Validation and Verification (V&V*) – рассмотрены в SWEBOK в рамках единой темы.
```
это сноска, почему удален символ?

js-denton replied:
По тексту это не сноска, а начало курсива. Начало которого было перенесено после символа тире.

js-denton replied:
Сноска получается на 432 строке. Переделаю на нормальную сноску.
В таком случае звёздочка на 423 является избыточно так как из-за неё строка превращалась в курсив

js-denton replied:
Исправлено 842fbf6

ligurio commented on 2_software_design.md:
```diff
@@ -587,10 +587,10 @@ languages, PDL)*: языки, используемые для описания 
 Хотя корни объектно-ориентированного проектирования лежат в абстракции данных
 (к которым добавлены поведенческие характеристики), так называемый
 responsibility-driven design или проектирование на основе <функциональной>
-ответственности по SWEBOK* может рассматриваться как альтернатива
```
Это сноска

js-denton replied:
Согласен, переделаю на нормальную сноску.

js-denton replied:
Исправлено 842fbf6

ligurio commented on software_lifecycle_models.md:
```diff
@@ -227,7 +227,7 @@ Processes” – “Процессы жизненного цикла прогр
             * задачи
 
-В общем случае, разбиение процесса базируется на широко распространенном PDCA-цикле:
+В общем случае, разбиение процесса базируется на широко распространённом PDCA-цикле:
```
если использовать ё, то везде по тексту. везде исправлено? если нет, то удалить изменение.

js-denton replied:
Возможно где-то ещё есть места. Ранее аналогичные исправления были в другом PR (к примеру есть в этой фиксации 4178591690dac9b6afc9ec367dbfde6af078203c).
Считаю такие исправления уместными.

ligurio replied:
>  Считаю такие исправления уместными.

тогда надо все сразу исправить

ligurio replied:
Строго говоря `е` вместо  `ё` это не опечатка. А многие издательства просто везде используют `е`.

js-denton replied:
Это плохо что кто-то вместо ё использует е. То что наш мозг начинает понимать слово не означает что это хорошо. иогурт вместо йогурт или ее вместо её не пишут.
Я исправил везде где мог - 9e67346c6337c20d99cac096b772f86e820c012e.

js-denton committed:
Updated footnotes

* Removed end-to-end numbering.
Now numbering refer to file where:
  first number - number document from name
  second number - ordinal number footnote
* Added two new footes - 10_1 and 2_3

ligurio commented on 2_software_design.md:
```diff
@@ -26,10 +26,10 @@ ISO/IEC (ГОСТ Р ИСО.МЭК) 12207. Проектирование прог
 рассматривать как деятельность, результат которой состоит из двух составных
```
в `0_introduction.md` не во всех терминах используется заглавная буква. Если исправлять, то везде.
В `10_software_quality.md` тоже. Вообщем надо весь текст смотреть и везде заменять.

js-denton replied:
Сверился с SWEBOK - удалил изменения.

ligurio commented on 2_software_design.md:
```diff
@@ -28,7 +28,7 @@ ISO/IEC (ГОСТ Р ИСО.МЭК) 12207. Проектирование прог
 
 - Архитектурный или высокоуровневый дизайн (Software Architectural Design,
```
> Какие знаки препинания ставятся после элементов перечня?

> После абзацев – элементов перечня ставится:

> а) запятая, если абзацы-элементы очень просты – из нескольких слов, без знаков препинания внутри, обозначены цифрой с закрывающей скобкой, строчной буквой с закрывающей скобкой, наборным знаком (например, тире) и начинаются со строчной буквы; но не будет ошибкой и постановка в подобных случаях после абзацев точки с запятой;

> б) точка с запятой, если элементы не совсем просты, со знаками препинания внутри, обозначены цифрой с закрывающей скобкой, строчной буквой с закрывающей скобкой, наборным знаком и начинаются со строчной буквы, т. е. строчной букве в начале элемента соответствует точка с запятой в конце его. Пример – настоящий перечень;

> в) точка, если элементы обозначены цифрой с точкой или прописной буквой с точкой. Точке после обозначения элемента соответствует точка в конце элемента.

https://gramota.ru/biblioteka/spravochniki/pismovnik/kak-oformlyat-perechni


Похоже, что точка с запятой здесь к месту.

js-denton replied:
Согласен. Удалил данную фиксацию.

ligurio commented on 6_software_configuration_management.md:
```diff
@@ -215,11 +215,23 @@ Identification)
 - Аудит конфигураций (Software Configuration Auditing)
 - Управление выпуском и поставкой (Software Release Management and Delivery)
```
исправлено, но не везде. как минимум в `0_introduction.md ` абзац не с длиной строк в 80 символов.

js-denton replied:
Исправил везде d03e68570deeea3803615ca221cb32f337adec0c
